guide/src/format/markdown.md

```diff
@@ -117,6 +117,28 @@ Which, of course displays the image like so:
 
 ![The Rust Logo](images/rust-logo-blk.svg)
 
+## Theme-dependent images
+
+Image variants for different themes can be used via CSS classes.
+
+Here is an example for different images for light vs. dark themes:
+
+```markdown
+<img src="images/rust-logo-blk.svg" class="no-dark-themes" />
+<img src="images/rust-logo-blk-dark.svg" class="no-light-themes" />
```

Comment:
I'd prefer `only-light-themes` and `only-dark-themes` for the class names ... since the negation is a bit unintuitive.

Reply:
Thanks for the review, you are probably right here. Will rework this accordingly.

Reply:
Technically, this fenced block is only `html` syntax.

Some MD linter prohibit use of raw HTML, so maybe change this to
~~~markdown
```markdown
![rust logo standard edition](images/rust-logo-blk.svg#only-light)
![rust logo dark mode](images/rust-logo-blk-dark.svg#only-dark)
```
~~~
as hayesall reported it working.